cypress/plugins/index.js

```diff
@@ -1,8 +1,8 @@
 const wp = require('@cypress/webpack-preprocessor');
 
-module.exports = on => {
+module.exports = (on) => {
   const options = {
-    webpackOptions: require('../../webpack.config')
+    webpackOptions: require('../../webpack.config'),
```

Comment:
You're right, it does look odd

Reply:
The plus is that it makes it easier to add another line, saving you the massive effort of typing in a comma on the previous line. And it does make the diffs a little cleaner. And easier to sort.

But yeah, it still looks weird.